src/main.jsx

```diff
@@ -67,6 +69,10 @@ const router = createBrowserRouter([
             path: "/confirmresetpassword",
             element: <ConfirmPasswordReset />,
           },
+          {
+            path: "/*",
+            element: <PageNotFound/>
+          },
```

Comment:
Question: I don't have experience with Browser Router, but I was wondering if this path of "/*" for the PageNotFound component was fine to use.